spec/mongo/crypt/auto_encrypter_spec.rb

```diff
@@ -157,6 +157,13 @@
         expect(auto_encrypter.metadata_client).not_to eq(client)
       end
     end
+
+    context 'when crypt shared library is available' do
+      it 'does not create a mongocryptd client' do
+        allow_any_instance_of(Mongo::Crypt::Handle).to receive(:"crypt_shared_lib_available?").and_return true
+        expect(auto_encrypter.mongocryptd_client).to be_nil
+      end
```

Comment:
While there is nothing wrong with this test, there should ideally be an integration test that asserts the client is functional, it doesn't spawn mongocryptd, and it doesn't wait for mongocryptd for 10 seconds.